index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="href='https://fonts.googleapis.com/css?family=Poppins" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+</head>
+
+<body>
+
+
+    <header>
+        <nav class="topnav">
+            <h2 class="logo">My Logo</h2>
+
+            <a class="icon">
+                <i class="fa fa-bars"></i>
+            </a>
+
+        </nav>
+    </header>
+    <section class="headline"></section>
+
+    <h1 class="name">
+        I'm Oluwatosin Festus Ajibola
+        Glad to meet you!
+    </h1>
+
+    <p class="words">
+        I'm a software developer! I can help you build a product,
+        feature or website. Look through some of my work and experience!
+        If you like what you see and have a project need coded, dont hesitate
+        to contact me.
+    </p>
```

Comment:
## Styling ✨✨

- [ ] Kindly make sure that your section uses the same **font** and **colors** provided by Figma design kindly remember using the same font and colors provided by Figma design in CSS helps to maintain consistency, design intent, branding, and accessibility. 😎😎😎

- [ ] I would be better also to use the same margin for your element spacing as that provided by the Figma design

### your design
![image](https://user-images.githubusercontent.com/102757126/212128471-b56e4ab3-dea5-462f-aa41-e4bd1f361991.png)

## Figma design
![image](https://user-images.githubusercontent.com/102757126/212128991-1167edb7-7206-4027-b06a-3855924bd6ba.png)